### DISCUSSION TITLE
Any way I can reduce the size of a chart ?

### DISCUSSION THREAD

**Discussion by StorageGuru** [RomRider/apexcharts-card#564] [Q&A]
I have a simple pie chart but is appears quite large. Any way I can reduce the size ? The following style section does not throw and arrow but also has no effect on the displayed size:

`type: custom:apexcharts-card
chart_type: pie
**style: |
  ha-card {
    width: 200px;
    height: 30px;
  }**
series:
  - entity: sensor.energie_bezug_total
    name: Energie Bezug Total
    group_by:
      func: sum
      duration: 1d
    color: red
  - entity: sensor.energie_einspeisung_total
    name: Energie Einspeisung Total
    group_by:
      func: sum
      duration: 1d
    color: green
`

**Comment by ThorAlex87** [comment]
You can try something like this:

apex_config:
  chart:
    height: 300px
    
I only set the height so the width can adapt to screen size.
